### PR TITLE
New version: Wagnardsoft.DisplayDriverUninstaller version 18.0.6.9

### PR DESCRIPTION
**Pull request by rkx0**
- [☑️] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [☑️ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [✖️] This PR only modifies one (1) manifest
- [☑️] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [☑️] Have you tested your manifest locally with `winget install --manifest <path>`?
- [☑️] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124331)

### CODE REVIEW

**Comment by cloudedaim** [comment]
@microsoft-github-policy-service agree

**Comment by cloudedaim** [comment]
Works fine on two of my PCs, one running Windows 10 and Windows 11. Might be a github server issue? Wagnard soft bans GitHub IPs from accessing their site, can that be the issue?

**Comment by cloudedaim** [comment]
> Hello @cloudedaim,
> 
> The package manager bot determined there was an issue with some of the URLs included in the manifest file. Please check the pull request for more details and make sure the urls are correct.
> 
> Template: msftbot/validationError/urls/smartScreen

The URLs are fine, and these URLs have been used in previous versions of the software as well. Checking the repo for  older versions of DDU reveals they're the same link, it's a problem with the testing bot.

File: manifests/w/Wagnardsoft/DisplayDriverUninstaller/18.0.6.9/Wagnardsoft.DisplayDriverUninstaller.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Wagnardsoft.DisplayDriverUninstaller
+PackageVersion: 18.0.6.9
+PackageLocale: en-US
+Publisher: Wagnardsoft
+PublisherUrl: https://www.wagnardsoft.com
+PrivacyUrl: https://wagnardsoft.com/content/Privacy-Policy
+PackageName: Display Driver Uninstaller
+PackageUrl: https://www.wagnardsoft.com/display-driver-uninstaller-ddu-
```
Comment (stephengillie) [inline manifests/w/Wagnardsoft/DisplayDriverUninstaller/18.0.6.9/Wagnardsoft.DisplayDriverUninstaller.locale.en-US.yaml:11]:
```suggestion
PublisherUrl: https://wagnardsoft.com
PrivacyUrl: https://wagnardsoft.com/content/Privacy-Policy
PackageName: Display Driver Uninstaller
PackageUrl: https://wagnardsoft.com/display-driver-uninstaller-ddu-
```

**Comment by cloudedaim** [comment]
Once again, it seems like an issue from the side of WagnardSoft. Checking the link shows it valid, are they blocking databases?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 11/1/2023 3:49:34 AM
Installer Verification Analysis Context Information: 
Error occurred while downloading installer from https://wagnardsoft.com/DDU/download/DDU v18.0.6.9_setup.exe. Forbidden (403). (0x80190193).
Error information: 
Http Response: Forbidden
Http Reason Phrase: Forbidden

**Comment by stephengillie** [comment]
> Once again, it seems like an issue from the side of WagnardSoft. Checking the link shows it valid, are they blocking databases?

Unfortunately, this appears to be the case.

**Comment by cloudedaim** [comment]
I have sent a support request to Wagnard, will comment back here with reply.

**Comment by Wagnard** [comment]
Hello,
I'm the owner of Wagnardsoft, it's hosted behind cloudflare, maybe it's considered by them as a bot or something like that.
Maybe I could whitelist the IP to help.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**